topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -98,15 +98,16 @@ private void createViews() {
 		injectApplauncherScript();
 		configureRootPanel();
 
-		String color = rootPanel.getElement().getAttribute(DATA_TMB_BGCOLOR);
+		String bg_color = rootPanel.getElement().getAttribute(DATA_TMB_BGCOLOR);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)